Makefile

```diff
@@ -531,7 +531,7 @@ generate-doctoc:
 	TRACE=$(TRACE) ./hack/generate-doctoc.sh
 
 .PHONY: generate-e2e-templates
-generate-e2e-templates: $(KUSTOMIZE) $(addprefix generate-e2e-templates-, v1.0 v1.4 v1.5 main) ## Generate cluster templates for all versions
+generate-e2e-templates: $(KUSTOMIZE) $(addprefix generate-e2e-templates-, v1.0 v1.5 v1.6 main) ## Generate cluster templates for all versions
```

Comment:
Not sure if this change will work without the corresponding changes to our tests that are using these files 
(please ignore if this PR is just WIP :))

Reply:
👋 yep just trying to get ahead 😄 .  will review more thoroughly once we have cut the `v1.6.0` release this week